grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html

```diff
@@ -0,0 +1,54 @@
+<h2>DESCRIPTION</h2>
+The script is intended to produce a set of SWD files as input to MaxEnt >= 3.3.3e using r.stats.<br>
+<br>
+The SWD file format is a simple CSV-like file file format as described in Elith et al. 2011.
+Generally it looks like:<br>
+<em>
+specie_name,X,Y,parameter_1,parameter_2,...<br>
+your_specie,1.1,1.1,2,4.7,...<br>
+</em>
+The first column always contains the name of the species, followed by
+two colums for the X- and Y-coordinates. Then each column
+represents one environmental parameter. In contrast to r.stats
+only integer values are accepted to represent NO DATA.<br>
+<br>
+A background SWD file is allways produced while specie output can be omitted.<br>
+<br>
+Multiple species can be processed, but each has to be in an
+individual raster map. Map names of the maps containing the environmental
+parameters can be replaced by short names, which should be used
+in MaxEnt 3.3.3.e.<br>
+<br>
+Results from MaxEnt can either be imported using r.in.xyz or
+calculated from MaxEnt lambdas file using the script
+r.maxent.lambdas.
```

Comment:
```suggestion
represents one environmental parameter. In contrast to <b>r.stats</b>
only integer values are accepted to represent NO DATA.<br>
<br>
A background SWD file is always produced while specie output can be omitted.<br>
<br>
Multiple species can be processed, but each has to be in an
individual raster map. Map names of the maps containing the environmental
parameters can be replaced by short names, which should be used
in MaxEnt 3.3.3.e.<br>
<br>
Results from MaxEnt can either be imported using <b>r.in.xyz</b> or
calculated from MaxEnt lambdas file using the script
<b>r.maxent.lambdas</b>.
```

(using bold for command names)